pkgs/top-level/all-packages.nix

```
@@ -2536,7 +2536,7 @@ in
 
   asciidoctor = callPackage ../tools/typesetting/asciidoctor {
     # kindlegen is unfree, don't enable by default
-    kindlegen = null;
+    kindlegen = if (config.allowUnfree) kindlegen else null;
```

```suggestion
    kindlegen = if (config.allowUnfree) then kindlegen else null;
```

I think this is the borg eval error but please test it.